litho-core/src/main/java/com/facebook/litho/DebugComponent.java

```diff
@@ -249,6 +249,17 @@ public String getTestKey() {
     return isLayoutNode() ? mNode.getTestKey() : null;
   }
 
+  /**
+   * @return This component's testTag or null if none is set. Unlike {@link getTestKey}, this
+   *     will return tags for any Component, including Components which are not LayoutNodes.
+   */
+  @Nullable
+  public Object getTestTag() {
+    Component component = mNode.getComponents().get(mComponentIndex);
+    CommonProps props = component.getCommonProps();
+    return props == null ? props.getTestTag();
```

Comment:
Should this be `return props != null ? props.getTestTag() : null;`?

Reply:
Yep, sorry, fixed that. I'm not sure how gradle was successfully running the tests with this.